x/airdrop/types/airdrop.go

```diff
@@ -2,6 +2,7 @@ package types
 
 import (
 	"fmt"
+	time "time"
```

Comment:
surprised this go through linting :/

Reply:
bad linter... auto import does that way too often for my liking...